src/vue/common/assets/AssetsList.vue

```diff
@@ -88,12 +111,38 @@ export default {
   methods: {
     selectAsset (asset) {
       this.selectedAsset = asset
+      this.isUpdateMode = false
       this.isDetailsDrawerShown = true
     },
-    updateSelectedAsset () {
+    refreshSelectedAsset () {
       this.selectedAsset = this.assetRecords
         .find(asset => asset.code === this.selectedAsset.code)
     },
+    async updateAsset () {
+      try {
+        const assetUpdateRequest = await this.fetchAssetUpdateRequest()
```

Comment:
why are you fetching `assetUpdate` request here, but `assetCreate` request inside the form itself? Doesn't them seem to be the similar actions?

Reply:
The `AssetForm` populating was completely rewritten. All the requests stuff is processing now inside the form itself. Also, a couple of comments were added for better understanding, why the specific actions are needed.

@max-nes @quinn71  please, take a look, say, what you think, and what we're gonna do with that stuff.

Reply:
lgtm